### ISSUE TITLE
Binance error -1100

### ISSUE THREAD

**Issue by alexeymarunin** [ccxt/ccxt#4360]
- OS: MacOS
- Programming Language version: Node.js v9.10.1
- CCXT version: 1.17.481
- Exchange: Binance
- Method: createOrder

Hi!

Error code [-1100](https://github.com/ccxt/ccxt/blob/78eec391d93b5f0401511c87b42589c36a69ced1/js/binance.js#L279) also returns when price in scientific format (like `1.7e-07`)
Look, I got this today
```
{
  "code":-1100,
  "msg":"Illegal characters found in parameter 'stopPrice'; legal range is '^([0-9]{1,20})(\\.[0-9]{1,20})?$'."}
```
I think we should try to convert price to fixed format string (like `0.00000017`) before requesting createOrder, and if it cannot be converted to numeric value will throw exception

**Comment by kroitor** [comment]
This was fixed in v1.17.557 (i'd recommend the most recent version), here: https://github.com/ccxt/ccxt/issues/3097. As you now we're under active development, so we ask our users to update frequently. Hope this answers the question.

**Comment by alexeymarunin** [comment]
@kroitor Thanks!
